daemon/mgr/spec_apparmor_linux_unsupported.go

```diff
@@ -0,0 +1,14 @@
+// +build !linux
+
+package mgr
+
+import (
+	"context"
+
+	specs "github.com/opencontainers/runtime-spec/specs-go"
+)
+
+// do nothing in this case
+func setupAppArmor(ctx context.Context, c *Container, s *specs.Spec) error {
+	return nil
```

Comment:
return `ErrUnsupportedArmor` error, if profile isn't empty. how do you think?

Reply:
For all apparmor setting, I leave a empty implement, consider it when we ready need for it. Or we can not think good enough.